app/javascript/app/data/data-explorer-constants.js

```diff
@@ -1,3 +1,5 @@
+import { TOP_EMITTERS } from './constants';
```

Comment:
Would it be better to import the `constants` file with `data/constants` instead that with the relative import? 
(just to avoid crashes in case the `data-explorer-constants` or `constants` files get moved to another folder).